docs/tutorials/own_machine/05_start_renku_session.rst

```diff
@@ -0,0 +1,81 @@
+.. _start_renku_session:
+
+Start a Renku session
+=====================
+
+#. **Navigate into your renku project directory:**
+
+   .. code-block:: shell-session
+
+       $ cd <project_name>
+
+
+#.  **Start a renku session:**
+
+    .. code-block:: shell-session
+
+       $ renku session start
+       ID          STATUS    URL
+       ----------  --------  ------------------------------------------------------------
+       f1693c198e  running   http://0.0.0.0:56674/?token=910ca732ef574049a22d41d0f1109f56
+
+    .. note::
+
+       Note: Renku may inform you that the container image for the current commit
+       does not yet exist on your machine, and Renku will offer to build it for you. Say yes!
+
+    .. note::
+
+        **Building too many images?**
+        By default, Renku builds a new image every time you start from a new commit.
```

Comment:
worth maybe noting that unless the environment is changing, the subsequent builds should be almost instantaneous.